### PR TITLE
Show the story bookend in vertical rendering.

### PR DESCRIPTION
**Pull request by gmajoulet**
Show the story bookend in vertical rendering.

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story.js [hunk 1]
```diff
@@ -1820,6 +1820,8 @@ export class AmpStory extends AMP.BaseElement {
           'amp-story-page amp-story-page-attachment'
         );
 
+        this.initializeBookend_().then(() => this.showBookend_());
```
Comment (Enriqe) [inline extensions/amp-story/1.0/amp-story.js]: Could having this outside of the `this.vsync_.mutate` below cause race conditions that would result in rendering the bookend sometimes before the page attachments, and sometimes after?

**Comment by Enriqe** [comment]
Do we want to render the bookend before the page attachments? I thought we wanted it at the end?

Also somewhat related: the page attachment in the bot view is showing multiple "x" buttons, we might want to fix that.

image

File: extensions/amp-story/1.0/amp-story.js (hunk 1 above)
Comment (gmajoulet) [reply extensions/amp-story/1.0/amp-story.js]: The `initializeBookend_` method creates the bookend synchronously, if it didn't exist yet. It'll always be before the page attachments

**Comment by gmajoulet** [comment]
Product decision was to render the page attachments last. I filed https://github.com/ampproject/amphtml/issues/22982 for the close buttons issue, that's specific to the visual diff tests

**Pull request merged**